_data/i18n/tables.yml

```diff
@@ -87,3 +87,9 @@ lightbox:
   beingUsed:
     en: 'Is being used by:'
     fr: 'Est utilisé par :'
+  website:
+    en: 'Website:'
+    fr: 'Site internet:'
+  githubRepo:
```

Comment:
Doesn't seem to be used anywhere? Also Github may not be the Repo used by any individual solution.

Reply:
@fnduister

Reply:
try I actually removed it for that reason, forgot to remove it from the table.

Reply:
i could use repo only but the information doesn't seem to be available for every software

Reply:
We don't ask for Repo at all anymore, so I suspect that's old data that never got scrubbed. I'd just remove it.

Reply:
it's removed